### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by ad9311**
## Greetings! 👋🏽 

In this PR I present the final stage of the Redux Bookstore project.
These are the changes so far:

- Apply styles to the different components.
- Book.js now has three children: BookData.js, BookCompletions.js and BookChapter.js

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @ad9311 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Navbar from './components/Navbar';
 import BookPage from './pages/BookPage';
 import CategoriesPage from './pages/CategoriesPage';
 import NoMatchPage from './pages/NoMatchPage';
+import './App.css';
```
Comment (mikenath223) [inline src/App.js:9]: - [ ] Kindly deploy the project online and include the live deploy link in the readme.

**Comment by ad9311** [comment]
As said by the previous reviewer I have deployed the app online and added the link in the README.
This is the link [Redux-Bookstore](https://epic-stonebraker-7bdd00.netlify.app/)

**Review by rukundoeric — APPROVED** [review]
Hi @ad9311,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**